vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/DatabaseWorkflow.java

```diff
@@ -430,7 +430,7 @@ public List<DatabaseOperation> phase(
       if (phase == Phase.FAILED) {
         target
             .consumableResources()
-            .forEach(cr -> cr.second().release(workflowName, workflowVersion, vidarrId));
+            .forEach(cr -> cr.second().release(workflowName, workflowVersion, vidarrId, Optional.empty()));
```

Comment:
So if a workflow run fails, it gets `set` in the WaitingState with a priority of 1? (input is not Present, so workflowPriority stays acceptedPriorities's min value)

Reply:
I saw this as being an extension where: 'this workflow run has failed, it no longer needs any resources, free them up but add it to the back of the wait list to retry later'

Reply:
FAILED jobs are basically dead to Vidarr until `attempt` is incremented, it doesn't need to stay in the waiting list, we can readd them if we choose to reattempt

Reply:
which should happen naturally as `request` would get called again, right?